redis/src/test/scala/zio/redis/ApiSpec.scala

```diff
@@ -268,6 +271,354 @@ object ApiSpec extends BaseSpec {
             } yield assert(hash)(equalTo(RedisType.Hash))
           }
         )
+      ),
+      suite("lists")(
+        testM("lIndex first element") {
```

Comment:
Please group the commands in subsuites as it is done for keys. This will make the test easier to read, and you will not need to repeat yourself in test descriptions.